code/game/gamemodes/cult/cult.dm

```diff
@@ -106,6 +106,7 @@
 //		grant_secondword(cult_mind.current)
 		update_cult_icons_added(cult_mind)
 		cult_mind.current << "<span class='notice'>You are a member of the cult!</span>"
+		cult_mind.current.verbs += /mob/living/proc/cult_innate_comm

```

Comment:
Huh... could've sworn I tested this, I'm sure its one of the reasons I coded the Debug-Game verb overriding round start checks.

We should probably make it so the initial set up uses the add_cultist() proc to prevent any other future weirdness.